packages/training/python/tests/integration/test_json_mode_integration.py

```
@@ -0,0 +1,554 @@
+        )
+        
+        degen_profit_inputs = TrajectoryRewardInputs(
+            final_pnl=50.0,
+            starting_balance=10000.0,
+            end_balance=10050.0,
+            format_score=0.7,
+            reasoning_score=0.6,
+        )
+        
+        active_degen_score = archetype_composite_reward(degen_loss_inputs, "degen", high_activity)
+        passive_degen_score = archetype_composite_reward(degen_profit_inputs, "degen", low_activity)
```

Ruff (0.14.10)

Local variable `passive_degen_score` is assigned to but never used

Remove assignment to unused variable `passive_degen_score`

(F841)